AnnoDesigner/Localization/TranslateExtension.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace AnnoDesigner.Localization
+{
+    public class TranslateKeyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var translations = value as IDictionary<string, string>;
+            var key = parameter as string;
+
+            if (translations.ContainsKey(key))
+            {
+                return translations[key];
+            }
+            return $"!{key}";
```

Comment:
Do we want to throw an exception here? Might be a more explicit way of warning us that something is wrong. At the very least, we could throw an exception when running a Debug build with `Debug.Assert` or `#if DEBUG`, and let this behaviour continue when in a Release.

Reply:
Good idea to throw an exception when in debug. But I'd argue to leave the prefix so that if some translation was missing in some language it sticks out more.